src/main/features/response/views/statement-of-means/employment/self-employment/on-tax-payments.njk

```diff
@@ -0,0 +1,44 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, textInput, poundInput, radioOption, textArea, saveAndContinueButton, errorSummary %}
+
+{% set heading = 'Are you behind on tax payments?' %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <form method="post" novalidate
+        class="analytics-submit-event-trigger"
+        data-event-action="SoM: What are you self-employed as"
+        data-event-label-from="declared">
```

Comment:
another GA label

Reply:
The same as above, this is just to extract value when even is triggered.